.github/workflows/build-test-push.yml

```diff
@@ -348,6 +347,181 @@ jobs:
             && ( test "${{ matrix.image_types }}" != "micro" && rpm -q gpg-pubkey) || true "
           done
 
+      -
+        name: Detect previous released tag for this image type
+        if: env.check_update != 0
+        id: detect-prev
+        env:
+          VMJ: ${{ matrix.version_major }}
+          IMT: ${{ matrix.image_types }}
+          VMIN: ${{ env.version_minor }}
+          DATE: ${{ env.date_stamp }}
+        run: |
+          IMAGE_REPO="${IMAGE_NAMES%%,*}"
+
+          # Fetch tag list and pick latest dated tag (<today): <VMJ><VMIN>-YYYYMMDD
```

Comment:
Why not just use the current `${VMJ}` tag, which should always be the latest released tag?